haskell-src-meta/src/Language/Haskell/Meta/Syntax/Translate.hs

```diff
@@ -811,4 +812,241 @@ instance ToDecs (Exts.Decl l) where
 instance ToDecs a => ToDecs [a] where
   toDecs a = concatMap toDecs a
 
+
+-----------------------------------------------------------------------------
+
+-- * Extensions
```

Comment:
nit: I might have put this in a separate file to keep the line count down, not a big deal.